db/migrate/20181003124257_add_uid_ems_to_physical_disks.rb

```diff
@@ -0,0 +1,5 @@
+class AddUidEmsToPhysicalDisks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_disks, :uid_ems, :string
```

Comment:
These don't need to be in separate migrations

Reply:
@agrare Changed it to be in only one migration.